pokemon-persistence/src/test/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dao/TrainerDAOTest.java

```diff
@@ -122,5 +122,17 @@ public void getAllTrainersTest() {
         assertThat(trainerDAO.getAllTrainers()).usingFieldByFieldElementComparator()
                 .containsOnly(ashTrainer, brockTrainer);
     }
+    
+    @Test
+    public void getAdminCountTestNoTrainer () {
+        assertThat(trainerDAO.getAdminCount()).isEqualTo(0);
+    }
+    @Test
+    public void getAdminCountTestOneAdmin () {
+        trainerDAO.createTrainer(ashTrainer);
+        trainerDAO.createTrainer(brockTrainer);
+        brockTrainer.setAdmin(true);
+        assertThat(trainerDAO.getAdminCount()).isEqualTo(1);
+    }
```

Comment:
Add empty line between test methods.
getAdminCountTestOneAdmin  - I do not think it works correctly because ashTrainer and brockTrainer is already created in setUp method. It would be better to create a new trainer who is admin and then call the assert.

Reply:
Empty line added

I don't see a reason why setting the admin flag on an existing user should not work but creating a new  "adminTrainer" does not do any harm. I have updated the code.